client/src/App.js

```diff
@@ -56,11 +59,37 @@ function App() {
         </nav>
       </header>
       <main>
-        <section className="hero">
-          <div className="container">
-            {/*<p className="heading-font">{!data ? "Loading..." : data}</p>{" "}*/}
+        <section className="hero my-44 flex flex-row px-10 w- auto h-auto pb-44">
+          <div className="container w-1/2 flex flex-col text-start justify-start items-start">
+            <img
+            alt="halogen"
+             className="w-3/4"
+             src="/images/title.png"/>
+            <p className="pt-7 mb-8 text-4xl leading-10">Instantly improve lighting in your video calls with halogen, a web-based lighting kit for all devices</p>
+            <div className="flex flex-col justify-start">
+              <LinearGradient width="400px" height="4px"></LinearGradient>
+              {/* could not apply margins to the components so had to create an empty p tag */}
+              <p className="mb-8"></p>
```

Comment:
Great workaround to get margin in here! I would maybe change this to a `div` since it won't contain any text though to make it more semantic